client/src/components/BottomNav/MenueButton.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { useState } from "react";
+import DiamondButton from "../../assets/mainButton.svg";
+import Overlay from "../../pages/OverlayMenue/Overlay";
+
+const MainButton = styled.button`
+  background: none;
+  border: none;
```

Comment:
I also like to set `outline: none` to get rid of the ugly blue outline when the button is focused/clicked

Reply:
thanks :)
Me too- I did that in my Globalstyle.js :)